airbyte-integrations/connectors/source-jira/source_jira/components/transformations.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+from dataclasses import InitVar, dataclass
+from typing import Any, Mapping, Optional
+
+from airbyte_cdk.sources.declarative.transformations import RecordTransformation
+from airbyte_cdk.sources.declarative.types import Config, FieldPointer, StreamSlice, StreamState
+
+
+@dataclass
+class RemoveEmptyFields(RecordTransformation):
```

Comment:
is there value in removing these fields? If anything, [we should go the other way around and make sure all fields are present ](https://airbytehq-team.slack.com/archives/C06FQEBNMEY/p1712778408904579)(doesn't have to be enforced now)

Reply:
I can't open the link, but I recall our discussion about this functionality. I mentioned that I would remove it if there is no reason to keep it as it is. However, I found this [PR](https://github.com/airbytehq/airbyte/pull/32927) which provides an explanation of why this feature was added